src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Recommended {
+
+  public static Drink getDrink(String storeid, String drinkName) {
```

Comment:
Google standard is camel case for Java i.e. storeId instead of storeid.

Reply:
fixed